NICE.Identity.Management/ClientApp/src/helpers/capitaliseWord.ts

```diff
@@ -0,0 +1,3 @@
+export const capitaliseWord = (word: string): string => {
```

Comment:
i'm not sure if this function is a good idea. why do we need it?

Reply:
I added it as the environment name from the mock api was all lowercase. I assumed this was the case with the proper api, but spotted it wasn't when you'd sent me the website json last week and I updated db.json